pkg/strategy/xgap/strategy.go

```diff
@@ -519,14 +544,32 @@ func (s *Strategy) placeOrders(ctx context.Context) {
 			Market:      s.tradingMarket,
 			TimeInForce: types.TimeInForceIOC,
 		},
+	} {
+		if s.tradingMarket.IsDustQuantity(order.Quantity, order.Price) {
```

Comment:
this check can be done before the loop?

Reply:
because the price and the quantity are the same

Reply:
Oh, you are right.